_automation_cookbook/create-comment.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Create a comment
+excerpt: Use record.create to make a comment
+summary: 
+layout: integration
+jumbotron:
+  title: Create a comment
+  breadcrumbs:
+    -
+      label: Resources &raquo;
+      url: /resources/
+    -
+      label: Automation Cookbook &raquo;
+      url: /resources/automation-cookbook/
+---
+You can use ```record.create``` to create a comment on any record.
```

Comment:
Inline code formatting only needs single backticks around the term.

```
`record.create:`
```

For consistency, include the `:` after command names.